ui/src/app/store/extension/extension.store.ts

```diff
@@ -17,4 +17,12 @@ export class ExtensionStore extends AbstractStore<
   protected get kind() {
     return 'Extension';
   }
+
+  public getUploadUrl() {
+    return this.service.getUploadUrl();
+  }
+
+  public importExtension(id: string) {
+    return this.service.importExtension(id);
+  }
```

Comment:
I'd annotate the methods with their returning types.

Reply:
Yup, will do...